src/runtime-rs/crates/hypervisor/src/device/driver/mod.rs

```diff
@@ -4,20 +4,204 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
+mod vfio;
 mod vhost_user;
 mod virtio_blk;
+mod virtio_fs;
+mod virtio_net;
+mod virtio_vsock;
+
+pub use vfio::{
+    bind_device_to_host, bind_device_to_vfio, get_vfio_device, HostDevice, VfioBusMode, VfioConfig,
+    VfioDevice,
+};
 pub use virtio_blk::{
     BlockConfig, BlockDevice, KATA_BLK_DEV_TYPE, KATA_MMIO_BLK_DEV_TYPE, VIRTIO_BLOCK_MMIO,
     VIRTIO_BLOCK_PCI,
 };
-mod virtio_net;
-pub use virtio_net::{Address, NetworkConfig, NetworkDevice};
-mod vfio;
-pub use vfio::{bind_device_to_host, bind_device_to_vfio, VfioBusMode, VfioConfig, VfioDevice};
-mod virtio_fs;
 pub use virtio_fs::{
     ShareFsDevice, ShareFsDeviceConfig, ShareFsMountConfig, ShareFsMountDevice, ShareFsMountType,
     ShareFsOperation,
 };
-mod virtio_vsock;
+pub use virtio_net::{Address, NetworkConfig, NetworkDevice};
 pub use virtio_vsock::{HybridVsockConfig, HybridVsockDevice, VsockConfig, VsockDevice};
+
+use anyhow::{anyhow, Context, Result};
+
+// Tips:
+// The Re-write `PciSlot` and `PciPath` with rust that it origins from `pcipath.go`:
+// https://github.com/kata-containers/kata-containers/blob/main/src/runtime/virtcontainers/types/pcipath.go
```

Comment:
Please do not use link from main branch, use it from specific commit.

Reply:
Thx, fixed